stdlib/manifest.py

```diff
@@ -81,6 +86,7 @@ def __init__(
         maintainer: str,
         licenses,
         upstream_url: str,
+        kind,
```

Comment:
Yes there is, to avoid adding `stdlib.kind` as an import of this module and therefore avoid the long term risk of having circle dependencies between modules induced by type hinting.

The type of the variable is still indicated in the documentation above.

Reply:
Ok, that's what I expected.

(However, given the content of the `kind` module, it would probably be okay to import it here, but I don't mind)